lib/bump/dependency_file.rb

```diff
@@ -1,15 +1,29 @@
 # frozen_string_literal: true
 module Bump
   class DependencyFile
-    attr_reader :name, :content
+    attr_reader :name, :content, :directory
 
-    def initialize(name:, content:)
+    def initialize(name:, content:, directory: "/")
       @name = name
       @content = content
+      @directory = directory
```

Comment:
Probably nicer to just do `@directory = clean_directory(directory)` rather than assigning then mutating.